katsdpimager/loader_katdal.py

```diff
@@ -187,16 +188,30 @@ def __init__(self, filename, options):
             corrections = 'none'
         _logger.info('Calibration corrections applied: %s', corrections)
 
-        # Determine RFI mask, if any (TODO: eventually katdal should do some of this)
+        # Determine RFI and band mask, if any. Using the band mask isn't
```

Comment:
On line 115 (parser for rfi-mask argument) we should probably mention that this now applies both the RFI and Band masks. I had first thought of splitting these into two arguments, but that seems overkill.

Reply:
Good point. Done, and also added to the user manual.